clang/include/clang/Basic/Version.h

```diff
@@ -56,6 +58,9 @@ namespace clang {
   /// for use in the CPP __VERSION__ macro, which includes the clang version
   /// number, the repository version, and the vendor tag.
   std::string getClangFullCPPVersion();
+
+  llvm::SmallVector<std::pair<llvm::StringRef, llvm::StringRef>, 2>
+  getSYCLVersionMacros(const LangOptions &LangOpts);
```

Comment:
Can you add a comment here (example line#57)?

Reply:
Thanks for the review! Done!

Reply:
Thanks Elizabeth.